src/server/api/routes/members.router.js

```diff
@@ -46,4 +46,54 @@ router.get('/', async (req, res) => {
   return res.json(membersOfBoard);
 });
 
+/**
+ * @swagger
+ * /users/{userId}/boards/{boardId}/members/{memberId}:
+ *  post:
+ *    tags:
+ *    - Members
+ *    summary: add a member to a board
+ *    description:
+ *      Will add a new member to a specific board.
+ *    produces: application/json
+ *    parameters:
+ *     - in: path
+ *       name: userId
+ *       schema:
+ *         type: integer
+ *         required: true
+ *         description: The userId of the owner of the board
+ *     - in: path
+ *       name: boardId
+ *       schema:
+ *         type: integer
+ *         required: true
+ *         description: The boardId of the specific board to get
+ *     - in: path
+ *       name: memberId
+ *       schema:
+ *         type: integer
+ *         required: true
+ *         description: The memberId of the user to add to the board as member
+ *
+ *    responses:
+ *      201:
+ *        description: Candidate added
+ *      5XX:
+ *        description: Unexpected error
+ *      400:
+ *          Invalid Id error
+ *      404:
+ *        description: Invalid entry error
+ */
+
+router.post('/:memberId', async (req, res) => {
+  await membersController.addMember(
+    req.params.userId,
+    req.params.boardId,
+    req.params.memberId,
+  );
```

Comment:
Why are we sending memberId and using /:memberId here? There is no memberId in db.

Reply:
Oh. I got it now. For adding the newMember with that memberId. However in that way I am able to put whatever number I want. Shouldn't it be the next number?